standards/application/telemetry.md

```diff
@@ -0,0 +1,39 @@
+---
+title: TELEMETRY
+name: Waku v2 Telemetry
```

Comment:
I'd propose to add `Metrics usage` section that align with our `building in public` principle. 
In that section we should explain what knowledge we are aiming to get from the collected data and re-assure a reader in us not having any way of personalizing collected data etc. This might come in handy in case we'd need to communicate is as a `Privacy policy`.

Reply:
I think beyond the metruc usage, we need to state what we are trying to learn so we can justify the work to ourselves. Not really any point collecting random data if you we don't know how we are going to analyse it.